settings.gradle

```diff
@@ -1,4 +1,5 @@
 include ':snowplow-tracker'
 include ':snowplow-demo-app'
```

Comment:
Nit: could we rename the `snowplow-demo-app` to `snowplow-demo-java` to make clear what is the difference with the Kotlin demo app?